prow/config/config_test.go

```diff
@@ -1079,8 +1084,9 @@ gerrit:
     - repo-b
 `,
 			expected: Gerrit{
-				TickInterval: &metav1.Duration{Duration: time.Minute},
-				RateLimit:    5,
+				TickInterval:              &metav1.Duration{Duration: time.Minute},
+				RateLimit:                 5,
+				AllowedPresubmitTriggerRe: &CopyableRegexp{},
```

Comment:
It is not expected for these to be empty, that is why the presubmits are failing. If there is some issue with a panic when you specify these we need to see that to debug it.

Reply:
We unexported the field `AllowedPresubmitTriggerRe`.